plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/ldap/LdapServerBackupForm.js

```diff
@@ -46,7 +46,11 @@ Ext.define('NX.coreui.view.ldap.LdapServerBackupForm', {
         var me = this,
           values = me.callParent(arguments);
 
-        values.backupMirrorEnabled = values.backupMirrorEnabled ? true : false;
+        if (values.backupMirrorEnabled === "on") {
```

Comment:
@adreghiciu Unlike checkboxes, there’s no config option to override the default values of a fieldset’s checkbox (e.g. inputValue and uncheckedValue). Any suggestions for how this could be made better?